Tests/SourceKitLSPTests/CodeActionTests.swift

```diff
@@ -1022,9 +1024,10 @@ final class CodeActionTests: XCTestCase {
             changes: [
               uri: [
                 TextEdit(
-                  range: positions["1️⃣"]..<positions["5️⃣"],
+                  range: positions["0️⃣"]..<positions["5️⃣"],
                   newText: ###"""
-                    ##"[\##(key): \##(d) \##(value)]"##
+                    
```

Comment:
Looks like this line has trailing whitespace. Could you run swift-format on your changes? https://github.com/swiftlang/sourcekit-lsp/blob/main/CONTRIBUTING.md#formatting

Reply:
Done, please test again.